styles/templates/_vowlClass.scss

```diff
@@ -0,0 +1,170 @@
+  background-color: #acf; 
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  overflow: hidden;
+  padding: 0px 0px 0px 10px;  
+  border-color: black; 
+  &:hover{
+    background-color: red;
+  };
+  &:focus{ 
+    border-color: red !important;
+    outline: none !important;
+  }
+}
+
+.ontodia-vowl-class-template_body__label {
+  font-size: 19px;
+  white-space: nowrap;
+  font-weight: 100;
+  margin-bottom: -7px;
+  width: 100%;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.ontodia-vowl-class-template_body_type-container {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  font-size: 11px;
+  font-style: italic;
+  color: black;
+  font-weight: 100;
+  display: flex;
+}
+
+.ontodia-vowl-class-template_body_type-container__type {
+  width: 100%;
+  flex-grow: 1;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.ontodia-vowl-class-template_body__icon {
+  position: inherit !important;
+  font-size: x-large;
+  margin: 5px 6px 5px 15px;
+  display: flex;
+}
+
+.ontodia-vowl-class-template_property {
+  border-radius: 10px;
+  background-color: whitesmoke;
+  margin-top: 5px;
+  border: 1px solid;
+  float: left;
+}
+
+.ontodia-vowl-class-template_property_content {
+  width: 100%;
+  height: 100%;
+  padding: 9px;
+}
+
+.ontodia-vowl-class-template_property_content_iri-line {
+  width: 100%;
+  overflow: hidden;
+  display: flex;
+}
+
+.ontodia-vowl-class-template_property_content_iri-line__label {
+  font-size: 12px;
+  font-weight: 100;
+  margin-right: 4px;
+  color: black;
+}
+
+.ontodia-vowl-class-template_property_content_iri-line__iri {
+  font-size: 12px;
+  width: 100%;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+.ontodia-vowl-class-template_property_content_iri-line__iri a {
+  color: black;
+}
+
+.ontodia-vowl-class-template_property_content__hr {
+  margin: 5px 0px 5px 0px;
+}
+
+.ontodia-vowl-class-template_property_content_property-table {
+  font-size: 15px;
+  font-weight: 100;
+  max-height: 200px;
+  overflow-y: scroll;
+  overflow-x: hidden;
+}
+
+.ontodia-vowl-class-template_property_content_property-table_row {
+  white-space: nowrap;
+}
+
+.ontodia-vowl-class-template_property_content_property-table_row__key {
+  display: inline-block;
+  width: 50%;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  vertical-align: top;
+}
+
+.ontodia-vowl-class-template_property_content_property-table_row_key {
+  display: inline-block;
+  width: 50%;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  vertical-align: top;
+}
+
+.ontodia-vowl-class-template_property_content_property-table_row_key_values {
+  display: inline-block;
+  width: 50%;
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
+
+.ontodia-vowl-class-template_property_content_property-table_row_key_values__value {
+  width: 100%;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: initial;
+  padding-right: 10px;
+}
+
+.link.joint-theme-default:hover {
```

Comment:
Put this style change into `CUSTOM_LINK_TEMPLATE` or `CUSTOM_LINK_TEMPLATE_SUBCLASSOF` instead of globally overriding style for all links.